charts/llm-d-modelservice/templates/_helpers.tpl

```diff
@@ -332,12 +332,17 @@ context is a dict with helm root context plus:
   {{- (include "llm-d-modelservice.hfEnv" .) | nindent 2 }}
   {{- with .container.livenessProbe }}
   livenessProbe:
-    {{- toYaml . | nindent 2 }}
+    {{- toYaml . | nindent 4 }}
   {{- end }}
   {{- with .container.readinessProbe }}
   readinessProbe:
-    {{- toYaml . | nindent 2 }}
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+  {{- with .container.startupProbe }}
+  startupProbe:
+    {{- toYaml . | nindent 4 }}
   {{- end }}
+  {{- with .container.ports }}
```

Comment:
The addition of `{{- with .container.ports }}` is incorrect.

Reply:
@kalantar thanks! I removed that line.